shared/js/initApp.js

```diff
@@ -150,7 +150,37 @@ $(window).on('function_editor_closed', function() {
   }
 })(appOptions.level);
 
+dashboard.saveProject = function(callback) {
+  var app_id = dashboard.currentApp.id;
+  dashboard.currentApp.startBlocks = Blockly.Xml.domToText(Blockly.Xml.blockSpaceToDom(Blockly.mainBlockSpace));
+  if (app_id) {
+    storageApps().update(app_id, dashboard.currentApp, function(data) {
+      console.log('Updated!');
```

Comment:
we can't have console.log because IE9 will crash if the console isn't open or something like that